### ISSUE TITLE
Unable to Inputs

### ISSUE THREAD

**Issue by gijo-varghese** [lusaxweb/vuesax#57]
OS: Ubuntu 18.04
NodeJS version: 10
Vuesax version: 3.0.12
Browser version: Chrome 66
Package manager: npm

I tried the following code
`<vs-input vs-label="Label" vs-placeholder="Placeholder" />`

and its throwing the following error:
```
[Vue warn]: Error in render: "TypeError: Cannot read property 'length' of undefined"

found in

---> <VsInput>
       <App> at src/App.vue
         <Root>
```

**Comment by gijo-varghese** [comment]
Ok. Figured it out, I had to put `v-model`. Otherwise it will throw error

**Comment by luisDanielRoviraContreras** [comment]
If exact but it would be better if the error were produced by vuesax requiring the v-model I will make that improvement thank you very much for contributing to vuesax

**Comment by gijo-varghese** [comment]
Thanks for your quick replies. For now I'm going to stick with Element UI. I really like your design. But lacks in the number of components. 

I'll try to contribute when I'm free

**Comment by luisDanielRoviraContreras** [comment]
Quiet yet we are in the process of creation and improvement of components any contribution is very grateful

**Comment by luisDanielRoviraContreras** [comment]
@gijo-varghese Ready friend already fixed the problem and gives a warn to report the lack of property thank you very much for the help 


corrected in the version v3.0.14

**Comment by gijo-varghese** [comment]
So fast!
